src/main.rs

```diff
@@ -60,22 +64,57 @@ pub enum Command {
     Tag(tag::Args),
 }
 
+/// Get the default araki settings
+fn default_settings() -> Result<ConfigBuilder<DefaultState>, ConfigError> {
+    Config::builder().set_default("backend", "github")
+}
+
+/// Get the araki configuration settings. In order, this merges
+///
+/// 1. Default settings
+/// 2. User-level araki.toml
+/// 3. Local araki.toml
+/// 4. Environment variables prefixed with 'ARAKI_'
+fn get_settings() -> Result<Config, ConfigError> {
+    let user_config = common::get_project_dir()
+        .map_err(|err| ConfigError::Message(format!("{err}")))?
+        .config_dir()
+        .join("araki")
+        .join("config");
```

Comment:
What path does this produce? From the [docs](https://crates.io/crates/directories), it looks like the result will be `<home dir>/.config/araki/araki/config`?